lambda/src/search-request-logger/search-request-logger.mjs

```diff
@@ -34,8 +35,13 @@ exports.handler = async event => {
       // API Gateway returns a 401 Unauthorized response without calling
       // the authorizer Lambda function.”
       // https://docs.aws.amazon.com/apigateway/latest/developerguide/configure-api-gateway-lambda-authorization-with-console.html"
-      Authorization:
-        (message.user && message.user.idToken) || 'unauthenticated',
+      Authorization: (messageUser && messageUser.idToken) || 'unauthenticated',
+      ...(messageUser.authUserTokens && {
+        'X-Tab-Auth-User-Tokens': messageUser.authUserTokens,
+      }),
+      ...(messageUser.authUserTokensSig && {
+        'X-Tab-Auth-User-Tokens-Sig': messageUser.authUserTokensSig,
+      }),
```

Comment:
Awaiting exploration of the requirements for the authorizer's identity sources before approving this. We may need to always send non-null values in the headers.

Reply:
Confirmed that the authorizer requires all identity sources to be set, so we'll have to take a different approach. Discussing elsewhere.